### PR TITLE
DNN-10324 Fixed issue in set-role command

### PR DESCRIPTION
**Pull request by mean2me**

### CODE REVIEW

File: src/Modules/Manage/Dnn.PersonaBar.Roles/Components/Prompt/Commands/SetRole.cs [hunk 1]
```diff
@@ -66,7 +66,7 @@ public override void Init(string[] args, PortalSettings portalSettings, UserInfo
                     break;
             }
 
-            if (string.IsNullOrEmpty(RoleName) && string.IsNullOrEmpty(Description) && !IsPublic.HasValue && !AutoAssign.HasValue)
+            if ((string.IsNullOrEmpty(RoleName) && RoleId < 0) && string.IsNullOrEmpty(Description) && !IsPublic.HasValue && !AutoAssign.HasValue && string.IsNullOrEmpty(status))
```
Comment (kanm2015) [inline src/Modules/Manage/Dnn.PersonaBar.Roles/Components/Prompt/Commands/SetRole.cs]: why check "status" here, seems not needed

Comment (mean2me) [reply src/Modules/Manage/Dnn.PersonaBar.Roles/Components/Prompt/Commands/SetRole.cs]: Hi Kan, I added this check to cover case like "set-role ROLE_ID --status Pending".
If I don't check for "status", that case will return an error.

Comment (kanm2015) [reply src/Modules/Manage/Dnn.PersonaBar.Roles/Components/Prompt/Commands/SetRole.cs]: status is not a mandatory param, it should still work if we missing it. I tested it a little bit and seems no error returned as long as I passed valid user Id. could you take a look again? thx

Comment (kanm2015) [reply src/Modules/Manage/Dnn.PersonaBar.Roles/Components/Prompt/Commands/SetRole.cs]: and for the case status not provided, the role will just stay on its current status

**Pull request closed without merging**